actions/test/action.yml

```diff
@@ -145,6 +145,27 @@ runs:
       with:
         +: git curl gnupg.org
```

Comment:
Before doing this, check if this required deps exist already or not (prefers using github script action and provided helper such as io.wich)